hkm/templates/hkm/snippets/cookie_consent.html

```diff
@@ -0,0 +1,50 @@
+{% load i18n %}
+
+<div id="cookie-consent">
+    <div id="cookie-consent__container">
+        <div id="cookie-consent__title-container">
+            <div id="cookie-consent__title">{% trans 'Helsinki photos uses cookies' %}</div>
+            <button type="button" id="cookie-consent__readmore-btn" class="cookie-consent__accordion-btn">
+                <span>{% trans 'Read more' %}</span>
+                <svg class="" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-label="angle-up" aria-hidden="true"><path fill-rule="evenodd" clip-rule="evenodd" d="M12 11.5L17 16.5L18.5 15L12 8.5L5.5 15L7 16.5L12 11.5Z" fill="currentColor"></path></svg>
+            </button>
+        </div>
+        <div id="cookie-consent__content">
+            <div id="cookie-consent__description">
+                {% trans 'This website uses required cookies' %}
+            </div>
+            <div id="cookie-consent__more-content__container">
+                <div id="cookie-consent__more-content__title">{% trans 'Cookie settings' %}</div>
+                <div class="cookie-consent__more-content__section">
+                    <span>{% trans 'About the cookies used on the website' %}</span>
+                    <p>{% trans 'The cookies used on the website'%}</p>
+                </div>
+                <div class="cookie-consent__more-content__section">
+                    <span>{% trans 'Matomo (optional)' %}</span>
+                    <p>{% trans 'Matomo description' %}</p>
+                    <table id="consent-table">
```

Comment:
A minor accessibility issue: "Add description to the table".
https://sonarcloud.io/project/issues?resolved=false&types=BUG&pullRequest=304&id=City-of-Helsinki_kuvaselaamo&open=AYuL6xbQR4bHoDNGSKbi